src/lib/components/User.svelte

```diff
@@ -13,22 +13,23 @@
   export let avatarRem = 1.5;
   export let noLink = false;
   export let showAddress = true;
-  export let address: string;
-  export let style: string = undefined;
+  export let address: string | undefined;
+  export let style: string | undefined = undefined;
   let hover = false;
 
   let firstRender = true;
 
-  $: ensName = $ensNames[address]?.name;
-  $: avatarUrl = $ensNames[address]?.pic;
-  $: toDisplay = ensName ? ensName : formatAddress(address.toLowerCase());
+  $: ensName = address && $ensNames[address]?.name;
```

Comment:
Not sure it's a problem here, but I've bumped into issues before with these assignments:
<img width="168" alt="Screenshot 2022-09-07 at 17 06 27" src="https://user-images.githubusercontent.com/158411/188912980-5b14b02c-7f84-48c8-b038-d09fc6893ac7.png">

Reply:
I believe it should be fine in this case, no way for `address` to be an empty string instead of undefined.